README.rst

```diff
@@ -3,4 +3,78 @@ Aexpect: Control your interactive applications
 
 This module provides services similar to the `pexpect` python library,
 so to speak, spawn and control interactive applications, such as `ssh`,
-`sftp` and others.
\ No newline at end of file
+`sftp` and others although it can be useful for pure streams as well.
+
+It's enhanced of multi-pattern matching, convenience features for
```

Comment:
"It's enhanced with" or if you want "It also offers" / "It also has the capabilities of"

Reply:
ack